internal/prune/prune.go

```diff
@@ -82,6 +82,7 @@ var DefaultFiles = []string{
 	".appveyor.yml",
 	"tsconfig.json",
 	"tslint.json",
+	"package-lock.json",
```

Comment:
npm always ignore this file https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files